cyhy_report/stakeholders/stakeholders.py

```diff
@@ -105,14 +116,18 @@ def generate_stakeholders_csv(db):
                 if org["_id"] in CI_sectors[sector]:
                     org_CI_sector = sector
                     break
-
+        
+        org_state = org["agency"]["location"].get("state", "N/A")
+        org_region = "Region {}".format(REGION_MAPPING.get(org_state, "N/A"))
+        
         wr.writerow(
             (
                 org["_id"],
                 org["agency"]["name"],
                 org["agency"]["location"]["name"].encode("utf-8"),
                 org["agency"]["location"]["county"],
                 org["agency"]["location"]["state"],
```

Comment:
Since we are now fetching the state above, you can do this:
```suggestion
                org_state,
```